english/marshmallow.md

```diff
@@ -2,6 +2,8 @@ Once upon a time many authors collaborated to make meaningless linked story.
 
 And that story was about [cricket](cricket/if-you-love-cricket/ifyoulovecricket.md)
 
+but some were interrupted by [Naruto](naruto/seventh-hokage.md) and the [Full Metal Alchemist](Full\Metal\Alchemist/fma.md) while
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.